app/routes/member/invites/$invitationId/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { LoaderArgs, redirect } from "@remix-run/node";
+import { requireMemberId, getMemberId } from "~/memberSession.server";
+import { getInvitation } from "~/models/invitation.server";
+import { useLoaderData, Link, Form, useTransition } from "@remix-run/react";
+import parseISO from "date-fns/parseISO";
+import format from "date-fns/format";
+import formatDistance from "date-fns/formatDistance";
+import { FaQuestion, FaCheck, FaExclamation } from "react-icons/fa";
+import { ImCross } from "react-icons/im";
+import { useState } from "react";
+
+export async function loader({ request, params }: LoaderArgs) {
+  await requireMemberId(request);
+
+  const invitationId = params["invitationId"];
+
+  if (!invitationId) {
+    throw redirect("/");
+  }
+
+  const invitation = await getInvitation(invitationId);
+
+  if (!invitation) throw redirect("/");
+
+  return invitation;
+}
+
+export default function MemberInvitationIdRoute() {
+  const invitation = useLoaderData<typeof loader>();
+  const transition = useTransition();
+  const numberOfGuests = invitation.guests ? invitation.guests : 0;
+
+  const [isEditing, setIsEditing] = useState(false);
+  const [answer, setAnswer] = useState("RESPONDED_MAYBE");
```

Comment:
We don't actually _need_ a `useState` at all here -- we can just use the built-in browser radio button state. If we remove this and change the 3 `checked={answer...` props below to `defaultChecked={` instead, that'll be simpler.